libs/damap/src/lib/components/dmp/repo/retention-period/retention-period.component.html

```diff
@@ -22,6 +26,9 @@
                 </mat-option>
               </mat-select>
             </mat-form-field>
+            <button mat-stroked-button class="unclickable">
```

Comment:
I would avoid using buttons as textboxes - this can be confusing, especially for users with disabilities. You could intead just keep blank text with some styling, or maybe create mat-cards, with the name as title. if the name is too long, you could truncate and show the rest via tooltip.

Reply:
Changed to using span tags, styled in a similar fashion:

![image](https://github.com/user-attachments/assets/da191a1a-9b33-4c9b-8585-39e9aafdf8a7)